INSTALL.md

```diff
@@ -4,13 +4,13 @@ The easiest and recommended way of installing the CLI on all platforms (Windows,
 to use the [Standalone Installation](#standalone-installation) described below. Some specific CLI
 commands have a few extra installation steps: see section [Additional Dependencies](#additional-dependencies).
 
-> **Windows users:** we now have a [YouTube video tutorial](https://www.youtube.com/watch?v=j3JoA1EINUA)
+> **Windows users:** We now have a [YouTube video tutorial](https://www.youtube.com/watch?v=j3JoA1EINUA)
```

Comment:
Note: the changes in this commit (https://github.com/balena-io/balena-cli/pull/1200/commits/a3c451eb424f264fe890e21c2f16935f3887569e) were requested by @hedss in a late review to #1175.